src/Sentry.Unity.Editor/SentryTestWindow.cs

```diff
@@ -14,7 +14,7 @@ public static SentryTestWindow Open()
         public void Dispose()
         {
             Close(); // calls 'OnLostFocus' implicitly
-            File.Delete(SentryUnityOptions.GetConfigPath(SentryOptionsAssetName));
+            // File.Delete(SentryUnityOptions.GetConfigPath(SentryOptionsAssetName));
```

Comment:
dead code. SHould we delete this?

Reply:
If this line is not used, than the class can be removed. The further call to `AssetDatabase.Refresh()` doesn't make sense too.